exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -272,6 +272,18 @@ def _translate_data(
             self._translate_events(sdk_span)
```

Comment:
I believe [proto](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto#L191) defines `dropped_attributes_count` nested inside `Event` and `Link` as well?

Reply:
I think this still applies?